plenum/common/messages/internal_messages.py

```diff
@@ -1,4 +1,14 @@
 from typing import NamedTuple, List
 
+
 ValidatorsChanged = NamedTuple('ValidatorsChange',
                                [('names', List[str])])
+
+ViewChangeInProgress = NamedTuple('StartViewChange',
+                                  [('in_progress', bool)])
+
+LedgerSyncStatus = NamedTuple('LedgerSyncStatus',
+                              [('is_synced', bool)])
+
+ParticipatingStatus = NamedTuple('LedgerParticipatingStatus',
```

Comment:
Do we actually need different statuses for LedgerSync and Participating?

Reply:
if we would pass status instead of 'is_synced' status, then we need to keep 'is_synced' defining logic for all of services (Ordering for now).